Helper/Order.php

```diff
@@ -796,77 +812,132 @@ private function checkTotalsMismatch($order, $transaction)
         $this->bugsnag->registerCallback(function ($report) use ($transaction, $cart) {
```

Comment:
do we already track Bolt transaction reference and magento order ID here? If not can you add?

If hook failed due to on-hold order, first thing oncall would do is to search bugsnag with reference